internal/zinc-apiinfo/src/main/scala/xsbt/api/Discovery.scala

```diff
@@ -25,33 +27,44 @@ class Discovery(baseClasses: Set[String], annotations: Set[String]) {
           Discovered.empty
       case _ => Discovered.empty
     }
-  def discover(c: ClassLike): Discovered =
-    {
-      val onClass = Discovery.findAnnotations(c.annotations, annotations)
-      val onDefs = Discovery.defAnnotations(c.structure, annotations) ++ c.savedAnnotations.filter(annotations)
-      val module = isModule(c)
-      new Discovered(bases(c.name, c.structure.parents), onClass ++ onDefs, module && hasMainMethod(c), module)
-    }
+  def discover(c: ClassLike): Discovered = {
+    val onClass = Discovery.findAnnotations(c.annotations, annotations)
+    val onDefs = Discovery.defAnnotations(c.structure, annotations) ++ c.savedAnnotations.filter(
+      annotations
+    )
+    val module = isModule(c)
+    new Discovered(
+      bases(c.name, c.structure.parents),
+      onClass ++ onDefs,
+      module && hasMainMethod(c),
+      module
+    )
+  }
 
   def bases(own: String, c: Seq[Type]): Set[String] =
     (own +: c.flatMap(simpleName)).filter(baseClasses).toSet
 
 }
 object Discovery {
-  def apply(subclasses: Set[String], annotations: Set[String])(definitions: Seq[Definition]): Seq[(Definition, Discovered)] =
-    {
-      val d = new Discovery(subclasses, annotations)
-      d(definitions)
-    }
+  def apply(subclasses: Set[String], annotations: Set[String])(
+    definitions: Seq[Definition]
+  ): Seq[(Definition, Discovered)] = {
+    val d = new Discovery(subclasses, annotations)
+    d(definitions)
+  }
   def applications(definitions: Seq[Definition]): Seq[(Definition, Discovered)] =
     apply(Set.empty, Set.empty)(definitions)
 
   def findAnnotations(as: Seq[Annotation], pred: String => Boolean): Set[String] =
-    as.flatMap { a => simpleName(a.base).filter(pred) }.toSet
+    as.flatMap { a =>
+      simpleName(a.base).filter(pred)
+    }.toSet
```

Comment:
Scalafmt is too eager at breaking down curly brace lambdas. I've known about this for a long time and iirc someone complained about it on gitter sometime but it has never been reported. I suspect it's not too hard to fix, so I invite you to open an issue ;)

However,  scalafmt will not try to move a curly brace lambda into a single line if there already is a newline inside the block.